tdrs-frontend/src/components/Welcome/Welcome.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import Button from '../Button'
+import { faSignOutAlt } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { useSelector } from 'react-redux'
+
+function Welcome() {
+  const logOut = () => {
+    window.location = `${process.env.REACT_APP_BACKEND_URL}/logout/oidc`
```

Comment:
This is the identical behavior of our other logout functions in the code base.

Reply:
Minor nit but if we have multiple functions doing this same thing it would be a good candidate for a utils function that could be imported.